ext/date/tests/timezone_location_get.phpt

```diff
@@ -15,7 +15,7 @@ array(4) {
   ["country_code"]=>
   string(2) "NO"
   ["latitude"]=>
-  float(59.91666)
+  float(59.9166%d)
```

Comment:
@nikic - Thanks for the quick look.  You're right of course.  This should be fixed now.